packages/lesswrong/lib/collections/comments/voting.ts

```diff
@@ -9,5 +9,10 @@ const customBaseScoreReadAccess = (user: DbUser|null, comment: DbComment) => {
 
 makeVoteable(Comments, {
   timeDecayScoresCronjob: true,
-  customBaseScoreReadAccess
+  customBaseScoreReadAccess,
+  userCanVoteOn: (user: DbUser|null, comment: DbComment, voteType: string) => {
```

Comment:
It's too much hassle to do in this PR but we should add a type for `voteType` (made a ticket https://app.asana.com/0/628521446211730/1203412948158482/f)